src/Predictors/MonteCarloPredictor.cpp

```diff
@@ -174,12 +174,12 @@ namespace PCOE {
                     // predicted values)
                     ++currentSavePt;
                     if (currentSavePt != savePts.end()) {
-                        timeOfCurrentSavePt = (*currentSavePt).time_since_epoch().count();
+                        timeOfCurrentSavePt = currentSavePt->time_since_epoch().count() / 100000;
                     }
                     auto z = model.getOutputVector();
```

Comment:
I think this is a bug in the predictor, and there is also bug/flaw in the battery model. I'm pretty sure we should be calling `model.outputEqn(...)` to get z, not getting an empty vector. Furthermore, the full body of Battery's `predictedOutputEqn` is

```
    // Return empty vector because there are no predicted outputs.
    return getPredictedOutputVector();
```